### ISSUE TITLE
'Our learning programmes' page

### ISSUE THREAD

**Issue by ShannonTrust** [yalla-coop/shannon-trust-website#134]
### Is your feature / client request related to a problem? Please describe.

We're going to start selling our numeracy resources soon, which means we need to do a bit of work on the website to reflect that we are offering both numeracy and reading. At the minute we have 3 pages (Turning Pages, Numeracy, Learn to read), that can all be merged into one page. By putting it all on one page, and in the navigation bar, we think it will create a much better user experience rather than continually linking to other pages to find out more. 

### Describe the solution you'd like

We have developed some new copy for a new page called 'our learning programmes'. It copies the same structure as the 'Turning Pages' page, so we could update that page with the new copy rather than creating a completely new page. However, that page is currently in the drop down menu and we would like it to replace 'learn to read' in the navigation bar. 

So in the navigation bar we would have: 
- What we do
- How you can help
- Our learning programmes (or learning programmes if that's too long)
- Contact

We would then want to delete the learn to read and numeracy pages. But if people have the links for these pages, could they be redirected to the 'our learning programmes' page, and the same with Turning Pages?

This is the updated copy:
[Our learning programmes.docx](https://github.com/user-attachments/files/16540182/Our.learning.programmes.docx)

Hopefully that makes sense, but let me know if you want a meeting to talk through it in more detail.

**Comment by cyberteenie** [comment]
Sounds good @ShannonTrust 
We will look through everything together and let you know if we have any questions

**Comment by ShannonTrust** [comment]
Hey @cyberteenie, do you have an idea of when this will be done, please? We're ready to go live with our new products, so it would be great if this could be done soon.

**Comment by AbdelhalimOJ** [comment]
Hi @ShannonTrust the page is ready here: https://www.shannontrust.org.uk/our-learning-programmes please check and let me know if there is anything.

CC @cyberteenie

**Comment by ShannonTrust** [comment]
Thanks @AbdelhalimOJ, this looks great. 

We would like this page to sit in the navigation bar and replace 'learning to read'. There might not be space for the full title, so we could just have 'our programmes' instead. 

We would then also like to delete the pages 'Turning Pages', 'Numeracy', and 'Learning to read'. But can these be redirected to the 'our learning programmes' page, please? People might have these saved for when they want to order our resources, so we don't want to potentially lose these customers. 

Any questions, let me know. Thank you.

**Comment by AbdelhalimOJ** [comment]
@ShannonTrust I added it to the navigation and drafted the other 3 pages. also setup a redirect as requested. Thanks!

**Comment by ShannonTrust** [comment]
Thank you, @AbdelhalimOJ. I'm just checking there's nothing else from the team, but if they're happy I'll close the issue. 

One thing from me though, I've noticed the quote carousel isn't coming up on the page. Looks like they're not showing up on any of the other pages either, could you look at this, please?

**Comment by AbdelhalimOJ** [comment]
Hi @ShannonTrust Do you mean these?
![image](https://github.com/user-attachments/assets/b13ef43e-88d2-4d0b-a398-f8c81232fd3d)

because I can still see the slider on "Our Learning Programmes" .. where else it should be?

**Comment by ShannonTrust** [comment]
@AbdelhalimOJ, yes, that's what I mean. That's weird, because it's not showing up for me. They're pretty much on every page, and it's showing up the same on the other pages as well.

![Screenshot (329)](https://github.com/user-attachments/assets/61e81d16-4618-463f-b007-c3ed212f9fcd)
![Screenshot (328)](https://github.com/user-attachments/assets/a7aef57f-1870-4e60-8d2a-9acd6436c768)

**Comment by cyberteenie** [comment]
@ShannonTrust from my side, I can also confirm seeing the carousels!
Can I suggest please clearing your cookie cache and seeing if they appear?

If still not, can you please make a quick video using Jam? This will help us see your device and browser settings to better debug: https://jam.dev/

Thanks!

**Comment by ShannonTrust** [comment]
Hi @cyberteenie, I've cleared cookies and cache and they're still not showing up. I've asked Amy and she can see them, so I'm not sure if it's just an issue with me - they're not showing up when I look on my phone either!  

I've recorded a video for you to see: https://jam.dev/c/b43f5d73-1cc6-4325-ac0a-320b07c9d70a

**Comment by cyberteenie** [comment]
Thanks for sharing this @ShannonTrust ! We will look into it!
@AbdelhalimOJ watching the jam, check out the console errors. I see a reference error, but not sure if it's related.

**Comment by AbdelhalimOJ** [comment]
Yes, the error is related. The sliders were built using Finsweet CMS Library, which has been replaced by Attributes as the CMS Library is no longer supported. I will update the sliders to use Attributes instead.

Can you check now if you are able to see it on all of the pages? @ShannonTrust

**Comment by ShannonTrust** [comment]
I can see them all now. Thank you, @AbdelhalimOJ and @cyberteenie for looking into this!
